### PR TITLE
Feedback from review

### PR DESCRIPTION
**Pull request by mimse**

### CODE REVIEW

File: libs/dh/wholesale/feature-request-calculation/src/lib/dh-wholesale-request-calculation.ts [hunk 1]
```diff
@@ -122,10 +125,10 @@ export class DhWholesaleRequestCalculationComponent {
   private _transloco = inject(TranslocoService);
   private _toastService = inject(WattToastService);
   private _destroyRef = inject(DestroyRef);
-  private _formGroupDirective = inject(FormGroupDirective);
-  private _selectedEicFunction: EicFunction | null | undefined = null;
+  private _selectedEicFunction!: EicFunction;
```
Comment (dzhavat) [inline libs/dh/wholesale/feature-request-calculation/src/lib/dh-wholesale-request-calculation.ts:128]: suggestion: the property can be `null`/`undefined` according to the return type, so let's reflect that here as well.

File: libs/dh/shared/ui-util/src/lib/dh-enum-to-dropdown-options.ts [hunk 2]
```diff
@@ -16,9 +16,24 @@
...
 import { WattDropdownOptions } from '@energinet-datahub/watt/dropdown';
 
-export function enumToDropdownOptions<T extends object>(enumObj: T) {
-  return Object.keys(enumObj).map((key) => ({
-    displayValue: key,
-    value: Object.values(enumObj)[Object.keys(enumObj).indexOf(key)],
-  })) as WattDropdownOptions;
+type Sort = 'asc' | 'desc';
+
+export function DhEnumToWattDropdownOptions<T extends object>(
```
Comment (dzhavat) [inline libs/dh/shared/ui-util/src/lib/dh-enum-to-dropdown-options.ts:21]: nitpick: throughout the project functions start with a small letter, so let's keep that convention.

File: libs/dh/shared/ui-util/src/lib/dh-enum-to-dropdown-options.ts [hunk 3]
```diff
@@ -16,9 +16,24 @@
  */
 import { WattDropdownOptions } from '@energinet-datahub/watt/dropdown';
 
-export function enumToDropdownOptions<T extends object>(enumObj: T) {
-  return Object.keys(enumObj).map((key) => ({
-    displayValue: key,
-    value: Object.values(enumObj)[Object.keys(enumObj).indexOf(key)],
-  })) as WattDropdownOptions;
+type Sort = 'asc' | 'desc';
```
Comment (dzhavat) [inline libs/dh/shared/ui-util/src/lib/dh-enum-to-dropdown-options.ts:19]: suggestion: this type can also be inlined next to the param.

File: libs/dh/shared/ui-util/src/lib/dh-enum-to-dropdown-options.ts [hunk 4]
```diff
@@ -16,9 +16,24 @@
...
+
+export function DhEnumToWattDropdownOptions<T extends object>(
+  enumObj: T,
+  exclude?: string[],
```
Comment (dzhavat) [inline libs/dh/shared/ui-util/src/lib/dh-enum-to-dropdown-options.ts:23]: nitpick: filter the unnecessary options out before calling the function. There's no need to create options just to be discarded in the next step.

Note: this comment is more of a personal preference (thus marked as "nitpick"). The current version is fine too. Ignore the comment if you prefer the current version.

Comment (mimse) [reply libs/dh/shared/ui-util/src/lib/dh-enum-to-dropdown-options.ts:23]: But you dont have a list before this function. You only have an enum. So it needs to be made into a list before the filter can be applied :-)

Comment (dzhavat) [reply libs/dh/shared/ui-util/src/lib/dh-enum-to-dropdown-options.ts:21]: Oh, yeah, that's true. Let's keep it then :)

**Review by dzhavat — APPROVED** [review]

**Pull request merged**